src/main/resources/db/migration/V5.3.0.6__2419-deactivate-users.sql

```diff
@@ -1,4 +1,6 @@
 -- Users can be deactivated.
 ALTER TABLE authenticateduser ADD COLUMN IF NOT EXISTS deactivated BOOLEAN;
+-- Prevent old users from having null for deactivated.
+UPDATE authenticateduser SET deactivated = false;
```

Comment:
@pdurbin should there be any concern at all if this is run on a db that already has deactivated users (if someone removes this from their flyway table, for example)? Maybe not, but might be safer to add a where clause OR as a default on the Alter table, i.e. 
```
ALTER TABLE authenticateduser ADD COLUMN IF NOT EXISTS deactivated BOOLEAN DEFAULT false
```

Reply:
either works for me. I think "WHERE deactivated IS NULL;" stands out more as intent.